framework/src/outputs/Output.C

```diff
@@ -260,6 +260,9 @@ Output::outputStep(const ExecFlagType & type)
   // Check whether we should output, then do it.
   if (shouldOutput())
   {
+    if (_problem_ptr->shouldPrintExecution(0))
+      _console << "[DBG]   Outputing " << _name << " at time " << _time << std::endl;
```

Comment:
Indentation is used to better illustrating that this is from the same `outputStep`. I applied the similar indentation to your other show executioner order places. Otherwise, it is a little hard to read the screen output.